Model/Cron.php

```diff
@@ -1190,13 +1199,12 @@ protected function _processNotification()
                  */
                 if (!$isOrderCc && strcmp($notificationPaymentMethod, $orderPaymentMethod) !== 0) {
```

Comment:
See my other comment, this could be changed to something like `if (!$isOrderCc && !$isOrderWallet && ...`